modules/services/event-bridge/variables.tf

```diff
@@ -15,7 +15,7 @@ variable "deploy_global_resources" {
   default     = false
 }
 
-variable "organization_units" {
+variable "org_units" {
```

Comment:
This will break the snippets in onboarding-api won't it?

Reply:
will create a FE ticket to change this.

Reply:
I can go ahead and add it my PR : https://github.com/draios/secure-backend/pull/16067/files

Reply:
@sameer-in But we need to make sure this is merged and published first, then I can merge the BE PR for release

Reply:
FE has pushed the change. so we are safe to merge.